### PR TITLE
added use reactivate feature

### PR DESCRIPTION
**Pull request by harshavardhan-sureify**
image

image

### CODE REVIEW

**Review by akhilarjun3-sureify — APPROVED** [review]

**Review by maniknr-sureify — APPROVED** [review]

File: src/components/admin/AllUsers.jsx [hunk 1]
```diff
@@ -45,7 +47,7 @@ const AllUsers = () => {
     const [filteredData, setFilteredData] = useState([]);
     const [page, setPage] = useState(2);
     const [rowsPerPage, setRowsPerPage] = useState(10);
-    const [deleteUser, setDeleteUser] = useState({});
+    const [statusUpdateUser, setStatusUpdateUser] = useState({});
```
Comment (abhishekdundigalla-sureify) [inline src/components/admin/AllUsers.jsx:50]: const [updateUserStatus, setUpdateUserStatus] = useState({});

File: src/components/admin/AllUsers.jsx [hunk 2]
```diff
@@ -66,22 +68,26 @@ const AllUsers = () => {
...
+    const handleStatusUpdateModal = (user) => {
+        setStatusUpdateUser(user);
         setIsOpen(true);
     };
     const handleChangePage = (event, newPage) => {
         setPage(newPage);
     };
 
-    const submitUserDelete = () => {
+    const submitUpdateUserStatus = () => {
```
Comment (abhishekdundigalla-sureify) [inline src/components/admin/AllUsers.jsx:79]: updateUserStatus or changeUserStatus
change it to any one of the above

File: src/components/admin/AllUsers.jsx [hunk 3]
```diff
@@ -309,8 +331,10 @@ const AllUsers = () => {
                     </Box>
                     <Box sx={{ typography: "subtitle2" }}>
                         <Typography variant="subtitle2">
-                            Are you sure want to delete the user
-                            {deleteUser.name}?
+                            {statusUpdateUser.is_active === "1"
```
Comment (abhishekdundigalla-sureify) [inline src/components/admin/AllUsers.jsx:334]: is it needed to check with "1" or just with 1
check once in the above part of logic too

**Review by abhishekdundigalla-sureify — CHANGES_REQUESTED** [review]
Please check comments

File: src/components/admin/AllUsers.jsx (hunk 1 above)
Comment (harshavardhan-sureify) [reply src/components/admin/AllUsers.jsx:50]: done

File: src/components/admin/AllUsers.jsx (hunk 2 above)
Comment (harshavardhan-sureify) [reply src/components/admin/AllUsers.jsx:79]: done

File: src/components/admin/AllUsers.jsx (hunk 3 above)
Comment (harshavardhan-sureify) [reply src/components/admin/AllUsers.jsx:334]: i checked the type of the is_active and it is string so i need to check with "1"

**Review by abhishekdundigalla-sureify — APPROVED** [review]

**Pull request merged**